src/ProgramBlockEditor.scss

```diff
@@ -73,56 +73,54 @@
     }
 }
 
-button.btn.ProgramBlockEditor__program-deleteAll-button {
+.ProgramBlockEditor__program-deleteAll-button {
```

Comment:
We can remove all the styles, except margin-left from the deleteAll-button, as its style is from IconButton.

Reply:
Including its focus style.

Reply:
This breaks the contrast styles, but I'll move the relevant stuff there.

Reply:
Huh, guess the contrast styles were broken before.  I'll fix those.